docs/core/build/xtask.md

```
@@ -0,0 +1,294 @@
+# Building with xtask
+
+`xtask` is the workspace automation tool for building, flashing and testing Trezor
+firmware. The central command is `xtask build`.
+
+## Quick start
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)